Resources/Prototypes/Entities/Objects/Misc/utensils.yml

```
@@ -178,6 +181,7 @@
     reactionTypes:
     - Stir
   - type: MeleeWeapon
+    animationRotation: 180 # WWDP
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)